develop/core/templates/core/account.html

```diff
@@ -18,8 +18,15 @@ <h4>{% trans 'Billing' %}</h4>
                         <div class="font-weight-bold">{% trans 'Payment Method' %}</div>
                         {% if payment %}
                         <div>
-                            {% blocktrans with account_type=payment.result.account_type account_number=payment.result.account_number %}{{ account_type }}
-                            ending in {{ account_number }}{% endblocktrans %}</div>
+                            {% blocktrans with account_type=payment.result.account_type account_number=payment.result.account_number %}
+                            {% if account_type %}
```

Comment:
I don't think you can use if statements inside blocktrans tags; you might need to do the if statement beforehand and use the result as your account_type variable within the blocktrans

Reply:
if the if/else isn't working maybe even try...
```html
 {% blocktrans with account_type=payment.result.account_type account_number=payment.result.account_number %}
 {{ { account_type|default:"Card" }} }} ending in {{ account_number }}
{% endblocktrans %}
```
just a thought...

Reply:
You can't do and if inside the transblock. I think @mgcamisa is on the right track

Reply:
Ended up doing this:
```
{% if payment.result.account_type %}
                        {% blocktrans with account_type=payment.result.account_type account_number=payment.result.account_number %}
                            {{ account_type }} ending in {{ account_number }} 
                        {% endblocktrans %}
                        {% else %}
                            {% blocktrans with account_number=payment.result.account_number %}
                                Card ending in {{ account_number }} 
                            {% endblocktrans %}
                        {% endif %}
```